packages/inline-edit/src/interfaces.ts

```diff
@@ -0,0 +1,90 @@
+import { ReactNode, RefObject } from 'react';
+
+
+import { ITheme, ICSSProperties, AllType } from '@xcritical/theme';
+
+
+export interface IInlineEditTheme extends ICSSProperties {
+  borderRadius?: number;
+}
+
+export type InlineEditTheme = ITheme<IInlineEditTheme>;
+
+export interface IFieldProps<T> {
+  value: T;
+  onChange: (e) => void;
+}
+
+export interface ICommonProps {
+  theme: InlineEditTheme;
+  appearance?: string;
+  baseAppearance?: string;
+  label?: string;
+  /** Set whether onConfirm should be called on blur. */
+  keepEditViewOpenOnBlur?: boolean;
```

Comment:
Looks like all this values unused.
- keepEditViewOpenOnBlur
- hideActionButtons
- isRequired
- onBlur
- onFocus